ext/opcache/Optimizer/sccp.c

```diff
@@ -1004,7 +1015,7 @@ static inline int ct_eval_func_call(
 				}
 			}
 			/* pass */
-		} else if (zend_string_equals_literal(name, "version_compare")) {
+		} else if (zend_string_equals_literal(name, "str_replace")) {
```

Comment:
Please at least add a TODO, I can imagine version_compare being important to performance in tight loops, e.g. to eliminate dead code paths

```
foreach ... {
   // more code
   if (version_compare(PHP_VERSION, '7.4.2', '<')) {
        // Some code path which creates/modifies a lot of local variables with unknown types
    }
}
```